lib/cryptoexchange/exchanges/cex/services/trades.rb

```diff
@@ -0,0 +1,32 @@
+module Cryptoexchange::Exchanges
+  module Cex
+    module Services
+      class Trades < Cryptoexchange::Services::Market
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Cex::Market::API_URL}/trade_history/#{market_pair.target}/#{market_pair.base}/?since=1"
```

Comment:
Do we need the `since=1`
It looks like that parameter is giving us old trades. Probably omit that if we want the latest current trades.

Reply:
`since=1` removed